src/data/achievements.test.js

```diff
@@ -43,3 +43,47 @@ describe('harvest-crop', () => {
     })
   })
 })
+
+const iAmRichVariants = [
+  ['i-am-rich-1', 500000, 'Earn $500,000', 'All sales receive a 5% bonus.'],
+  ['i-am-rich-2', 1000000, 'Earn $1,000,000', 'All sales receive a 10% bonus.'],
+  [
+    'i-am-rich-3',
+    1000000000,
+    'Earn $1,000,000,000',
+    'All sales receive a 25% bonus.',
```

Comment:
not sure how you feel about these kinda copy tests, they could also be snapshots @jeremyckahn . i kept manually testing that the copy was correct though so it felt right to add an automated test for it

Reply:
sounds good to me!